ipatests/test_integration/test_cert.py

```diff
@@ -168,6 +168,119 @@ def test_ipa_getcert_san_aci(self):
         ipaddrs = ext.value.get_values_for_type(x509.IPAddress)
         assert ipaddrs == [ipaddress.ip_address(self.clients[0].ip)]
 
+    def test_service_del_revocation(self):
+        """Test that removing a service revokes its certificate
+
+           This removes the service created in the previous test
+           and ensures that other certificates are not revoked when
+           one is.
+        """
+        hostname = self.clients[0].hostname
+        certfile = os.path.join(paths.OPENSSL_CERTS_DIR, "testservice.pem")
```

Comment:
The file is called test2.pem in the previous test

Reply:
Gah, I fixed then reverted this for some unknown reason. Fixed again.